include/utility/dbus_utility.hpp

```diff
@@ -197,5 +197,39 @@ inline bool callPIM(types::ObjectMap&& objectMap)
     }
     return true;
 }
+
+/**
+ * @brief API to check D-Bus service status.
+ *
+ * Any failure in calling the method "NameHasOwner" implies that the service is
+ * not in a running state. Hence the API returns false in case of any exception
+ * as well.
+ *
+ * @param[in] i_serviceName - D-Bus service name whose status is to be checked.
+ * @return bool - True if the service is running false otherwise.
+ */
+inline bool checkServiceStatus(const std::string& i_serviceName)
```

Comment:
isServiceRunning() could be the better name for this method, as it returns bool.

Reply:
Done.